sdk/ml/azure-ai-ml/tests/compute/unittests/test_compute_entity.py

```diff
@@ -140,6 +140,7 @@ def test_compute_instance_schedules_from_yaml(self):
         assert compute_instance2.schedules.compute_start_stop[1].trigger.frequency == "week"
         assert compute_instance2.schedules.compute_start_stop[1].trigger.interval == 1
         assert compute_instance2.schedules.compute_start_stop[1].trigger.schedule is not None
+        assert compute_instance2.idle_time_before_shutdown_minutes == 15
```

Comment:
Can we please add a test which validated the conversion to rest object and if this property gets translated as expected by rest api ?

Reply:
Added a line into this test to validate the number was converted to the expected ISO time format.